src/shared/components/DurationInput.tsx

```diff
@@ -41,6 +42,7 @@ const DurationInput: FC<Props> = ({
   customClass,
   dividerText,
   dividerOnClick,
+  metric,
```

Comment:
It seems like the eventing is only happening in one place at a parent level, why not set that stuff in this component rather than prop drilling the metric?

Reply:
This component is not specific to the dashboard auto refresh flow. This may be the only place we are using this metric prop at the moment, but it prevent tying the behavior to this feature

Reply:
Gotcha. Makes sense I guess